pandas/io/tests/parser/test_unsupported.py

```diff
@@ -20,6 +20,16 @@
 
 
 class TestUnsupportedFeatures(tm.TestCase):
+    def test_mangle_dupe_cols_false(self):
+        # see gh-12935
+        data = 'a b c\n1 2 3'
+        msg = 'is not supported'
+
+        for engine in ('c', 'python'):
+            with tm.assertRaisesRegexp(ValueError, msg):
```

Comment:
No.  As of now, it is an unsupported feature because it fails with a duplicate header (or `names`) and when you set `mangle_dupe_cols=False`, so it is treated like any other unsupported feature in `parser.py` - we just don't allow it period.